docs/pages/get-started/lexical-react.mdx

```diff
@@ -0,0 +1,232 @@
+
+      ```tsx file="App.tsx" highlight="11-15"
+      "use client";
+
+      import {
+        LiveblocksProvider,
+        RoomProvider,
+      } from "@liveblocks/react/suspense";
+      import { Editor } from "./Editor";
+
+      export default function App() {
+        return (
+          <LiveblocksProvider publicApiKey={"{{PUBLIC_KEY}}"}>
+            <RoomProvider id="my-room" initialPresence={{}}>
+              {/* ... */}
+            </RoomProvider>
+          </LiveblocksProvider>
+        );
+      }
+      ```
+
+    </StepContent>
+
+  </Step>
+  <Step>
+    <StepTitle>Join a Liveblocks room</StepTitle>
+    <StepContent>
+
+      After setting up the room, you can add collaborative components inside it, using
+      [`ClientSideSuspense`](/docs/api-reference/liveblocks-react#ClientSideSuspense) to add loading spinners to your app.
+
+      ```tsx file="App.tsx" highlight="14-16"
+      "use client";
+
+      import {
+        LiveblocksProvider,
+        RoomProvider,
+        ClientSideSuspense,
+      } from "@liveblocks/react/suspense";
+      import { Editor } from "./Editor";
+
+      export default function App() {
+        return (
+          <LiveblocksProvider publicApiKey={"{{PUBLIC_KEY}}"}>
+            <RoomProvider id="my-room" initialPresence={{}}>
+              <ClientSideSuspense fallback={<div>Loading…</div>}>
+                <Editor />
+              </ClientSideSuspense>
+            </RoomProvider>
+          </LiveblocksProvider>
+        );
+      }
+      ```
+
+    </StepContent>
+
+  </Step>
+  <Step>
+    <StepTitle>Set up the collaborative Lexical text editor</StepTitle>
+    <StepContent>
+
+      Now that we set up Liveblocks, we can start integrating Lexical and Liveblocks in the `Editor.tsx` file.
+      To make the editor collaborative, we can use
+      [`LiveblocksPlugin`](http://localhost:3001/docs/api-reference/liveblocks-react-lexical#LiveblocksPlugin)
+      from `@liveblocks/lexical-react`. [`FloatingComposer`](/docs/api-reference/liveblocks-react-lexical#FloatingComposer)
+      and [`ThreadPanel`](/docs/api-reference/liveblocks-react-lexical#ThreadPanel) add
+      [Comments](/docs/products/comments) to your text editor, and are both optional.
+
+
+      ```tsx file="Editor.tsx"
+      "use client";
+
+      import { LexicalComposer } from "@lexical/react/LexicalComposer";
+      import { RichTextPlugin } from "@lexical/react/LexicalRichTextPlugin";
+      import { ContentEditable } from "@lexical/react/LexicalContentEditable";
+      import { LexicalErrorBoundary } from "@lexical/react/LexicalErrorBoundary";
+      import {
+        liveblocksConfig,
+        LiveblocksPlugin,
+        FloatingComposer,
+        ThreadPanel,
+      } from "@liveblocks/react-lexical";
+
+      export default Editor() {
+        const room = useRoom();
+
+        // Wrap your initial config with `liveblocksConfig`
+        const initialConfig = liveblocksConfig({
```

Comment:
Should we also highlight the fact that users are expected to wrap their lexical configuration within `liveblocksConfig`? This function ensures `ThreadMarkNode` and `MentionNode` are correctly added to the editor config.